src/App.tsx

```diff
@@ -1,204 +1,198 @@
-              />
-            </label>
-
-            <span data-cy="TodoTitle" className="todo__title">
-              Todo is being saved now
-            </span>
-
-            <button type="button" className="todo__remove" data-cy="TodoDelete">
-              ×
-            </button>
-
-            {/* 'is-active' class puts this modal on top of the todo */}
-            <div data-cy="TodoLoader" className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-        </section>
+        <TodoList todos={filteredTodos} handleTodoClick={handleTodoClick} />
 
         {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer" data-cy="Footer">
-          <span className="todo-count" data-cy="TodosCounter">
-            3 items left
-          </span>
-
-          {/* Active link should have the 'selected' class */}
-          <nav className="filter" data-cy="Filter">
-            <a
-              href="#/"
-              className="filter__link selected"
-              data-cy="FilterLinkAll"
-            >
-              All
-            </a>
 
-            <a
-              href="#/active"
-              className="filter__link"
-              data-cy="FilterLinkActive"
-            >
-              Active
-            </a>
+        {!!todos.length && (
+          <footer className="todoapp__footer" data-cy="Footer">
+            <span className="todo-count" data-cy="TodosCounter">
+              {todos.filter(todo => !todo.completed).length} items left
+            </span>
 
-            <a
-              href="#/completed"
-              className="filter__link"
-              data-cy="FilterLinkCompleted"
+            {/* Active link should have the 'selected' class */}
+            <nav className="filter" data-cy="Filter">
+              <a
+                href="#/"
+                className={cn('filter__link', {
+                  selected: selectedFilter === 'All',
+                })}
+                data-cy="FilterLinkAll"
+                onClick={() => setSelectedFilter(Filter.ALL)}
+              >
+                {Filter.ALL}
+              </a>
+
+              <a
+                href="#/active"
+                className={cn('filter__link', {
+                  selected: selectedFilter === 'Active',
+                })}
+                data-cy="FilterLinkActive"
+                onClick={() => setSelectedFilter(Filter.ACTIVE)}
+              >
+                {Filter.ACTIVE}
+              </a>
+
+              <a
+                href="#/completed"
+                className={cn('filter__link', {
+                  selected: selectedFilter === 'Completed',
+                })}
+                data-cy="FilterLinkCompleted"
+                onClick={() => setSelectedFilter(Filter.COMPLETED)}
+              >
+                {Filter.COMPLETED}
```

Comment:
Still not fixed from the previous review,
<img width="957" alt="image" src="https://github.com/mate-academy/react_todo-app-loading-todos/assets/64750958/b2e78581-46ca-4604-ba8b-0f695fb7cdbf">
```suggestion
             Object.keys(Filter).map(filter => (
               .......
             ))
```